frontend/src/bundles/auth/pages/auth.tsx

```diff
@@ -71,7 +80,7 @@ const Auth: React.FC = () => {
             <div className={getValidClassNames(classes.base, classes.form)}>
                 {getScreen(pathname)}
             </div>
-            <div className="flex flex-1 items-center justify-center text-xl text-white">
+            <div className="hidden flex-1 items-center justify-center text-xl text-white lg:flex">
```

Comment:
keep class names in the groups
`hidden lg:flex .... bg-red lg:bg-blue .....` etc.

Reply:
I guess this is something related to Prettier because it changes its position when I save it